/*
try{
code which may give error 
}
catch(error){
handle error
}
*/
try{

console.log("hlo");
}
catch(error){
console.log("form catch");
console.log(" ")